koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -480,10 +469,15 @@ def check_whitelist(self, name, target_info):
             raise koji.BuildError("package (container)  %s is blocked for tag %s" %
                                   (name, target_info['dest_tag_name']))
 
-    def upload_build_annotations(self, annotations):
-        whitelist_str = annotations.get('koji_task_annotations_whitelist', "[]")
-        whitelist = json.loads(whitelist_str)
-        task_annotations = {k: v for k, v in annotations.items() if k in whitelist}
+    def upload_build_results_as_annotations(self, build_results):
+        task_annotations = {
+            # TODO: previously, remote_sources was double JSON-encoded, do we need to keep that?
+            #   prev: {"remote_sources": "[{\\"name\\": null, \\"url\\": \\"some url\\"}]"}
+            #   curr: {"remote_sources": [{"name": null, "url": "some url"}]}
```

Comment:
How much do we need to worry about keeping build_annotations.json exactly the same as before? I'm thinking not that much, since when we last switched from `remote_source_url` to `remote_sources`, nobody seemed to notice

Reply:
On the other hand, it's easier to preserve the format than to answer this question. I'll change this to match the previous format